apps/ledger-live-desktop/src/renderer/components/EarnRewardsInfoModal.jsx

```diff
@@ -84,7 +84,7 @@ export default function EarnRewardsInfoModal({
               <Button ml={2} secondary onClick={onClose}>
                 <Trans i18nKey="common.cancel" />
               </Button>
-              <Button ml={2} primary onClick={onNext}>
+              <Button ml={2} primary onClick={onNext} data-test-id="modal-continue-button">
```

Comment:
Is this a generic button or one specific to staking? If it's specific maybe call it `earn-rewards-modal-continue-button` to make it more obvious what it's doing

Reply:
I think it is simpler use always the same data-test-id here, it is simpler to access to the modal and you do not need recreate a Model